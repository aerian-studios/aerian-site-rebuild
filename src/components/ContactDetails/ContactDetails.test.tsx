import * as React from "react";

import * as renderer from "react-test-renderer";

import { ContactDetails } from "./index";

describe("ContactDetails", () => {
    it("renders correctly", () => {
        const tree = renderer
            .create(
                <ContactDetails
                    className="myClass"
                    phoneNumber="0345 408 6009"
                    street="The Old Malthouse, Mill Lane"
                    locality="Box"
                    region="Wiltshire"
                    postcode="SN13 8PN"
                />
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
