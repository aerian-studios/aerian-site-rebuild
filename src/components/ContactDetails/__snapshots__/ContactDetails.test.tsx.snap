// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactDetails renders correctly 1`] = `
<div
  className=" myClass"
  itemscope={true}
  itemtype="http://schema.org/Organization"
>
  <p>
    Call 
    <span
      itemprop="telephone"
    >
      0345 408 6009
    </span>
  </p>
  <p
    itemprop="address"
    itemscope={true}
    itemtype="http://schema.org/PostalAddress"
  >
    <span
      itemprop="streetAddress"
    >
      The Old Malthouse, Mill Lane
    </span>
    <span
      itemprop="addressLocality"
    >
      Box
    </span>
    <span
      itemprop="addressRegion"
    >
      Wiltshire
    </span>
    <span
      itemprop="postalCode"
    >
      SN13 8PN
    </span>
  </p>
</div>
`;
